src/pages/Confirmation.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect} from 'react';
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+import { connect } from "react-redux";
+import { fetchAllApps } from "../redux/actions/fetchApps";
+
+// for this to work, in student data, we need to include the apps data id they are applying to
+
+const Confirmation = ({apps, getAllApps, appId}) => {  
+  useEffect(() => {
+        getAllApps();
+    }, [getAllApps]);
```

Comment:
I think maybe this should just be an empty array, as the function itself will not change, so it'll still run only once.